cluster-provision/gocli/go.mod

```diff
@@ -21,3 +21,5 @@ require (
 )
 
 replace github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.4.1
+
+go 1.13
```

Comment:
Is this intentional or a leftover?

Reply:
will need to ask @rmohr here
(or try and error)